edx_proctoring/tests/test_api.py

```diff
@@ -873,6 +897,44 @@ def test_get_studentview_started_exam(self):  # pylint: disable=invalid-name
         )
         self.assertIsNone(rendered_response)
 
+    def test_get_studentview_started_practice_exam(self):  # pylint: disable=invalid-name
```

Comment:
please remove this line `# pylint: disable=invalid-name`  from here and look for others too.
we had already disabled the `# pylint: disable=invalid-name` in the top of the `test_api` file.